docs/fields.asciidoc

```diff
@@ -328,6 +330,20 @@ Agent version.
 Error-specific data for APM
 
 
+[float]
+=== `view errors`
+
+type: keyword
+
+format: url
+
+[float]
+=== `error id icon`
```

Comment:
I wonder if we should have a special note on these fields that these are script field. This is a change that would require to our docs script afterwards.

Reply:
We could add something to the docs script as soon as there is a `script` set.